src/components/JobCard.js

```diff
@@ -3,57 +3,74 @@ import { Row, Col, Badge } from "react-bootstrap";
 import moment from "moment";
 import { useHistory } from "react-router-dom";
 
-export default function JobCard(props) {
-  let item = props.job;
+/**
+Sample prop job:
+{
+    id: 0,
+    img: "https://cdn.itviec.com/employers/antada/logo/s65/re9ZtCRGvxqdPL8otJ8Ca8zR/ANTADA%20-%20LOGO_final.png",
```

Comment:
in our json, we should probably upload the images somewhere else btw. these images will break over time, you can kinda tell by the URL that it's a temporary url.

Reply:
i saved the logo images into src/images folder